index.html

```diff
@@ -1,534 +1,1010 @@
-          Sebastian a primit numeroase premii și distincţii. S-a remarcat în
-          mod deosebit prin implicarea sa în cadrul proiectului “Educație
-          juridică în școli”, demonstrându-și abilitatea de a explica și
-          dezbate conceptele juridice într-un mod accesibil pentru elevi.
-        </p>
-      </div>
-      <div class="operations__content operations__content--3">
-        <div class="operations__icon operations__icon--3">
-          <svg>
-            <use xlink:href="img/icons.svg#icon-user-x"></use>
-          </svg>
-        </div>
-        <h5 class="operations__header">
-          Pasiune pentru justiție de calitate
-        </h5>
-        <p>
-          Pasiunea pentru un act de justiție de calitate l-a împins pe
-          Sebastian Nagy sa fie prezent în mod constant în presa locală și
-          națională, contribuind cu expertiza sa la educația juridică a
-          intregii societati.
-        </p>
+    <section class="section" id="despre">
+      <div class="section__title">
+        <h2 class="section__description">Despre</h2>
+        <h3 class="section__header">Abordarea noastră în domeniul dreptului</h3>
       </div>
-    </div>
-  </section>
 
-  <section class="section" id="servicii">
-    <div class="section__title">
-      <h2 class="section__description">Soluții există</h2>
-      <h3 class="section__header">Servicii juridice de încredere în Cluj-Napoca</h3>
-    </div>
-
-    <div class="features">
-      <img src="img/law-1--lazy.jpg" data-src="img/law-1.jpg" alt="Computer" class="features__img lazy-img" />
-      <div class="features__feature">
-        <div class="features__icon">
-          <svg>
-            <use xlink:href="img/icons.svg#icon-monitor"></use>
-          </svg>
+      <div class="operations">
+        <div class="operations__tab-container">
+          <button class="btn operations__tab operations__tab--1" data-tab="1">
+            Determinare
+          </button>
+          <button class="btn operations__tab operations__tab--2" data-tab="2">
+            Abilitate
+          </button>
+          <button
+            class="btn operations__tab operations__tab--3 operations__tab--active"
+            data-tab="3"
+          >
+            Pasiune
+          </button>
         </div>
-        <h5 class="features__header">Drept Penal</h5>
-        <p>
-          Apărare și reprezentare în dosare penale, asigurându-se că
-          drepturile și interesele clienților săi sunt protejate cu fermitate.
-        </p>
-      </div>
-
-      <div class="features__feature">
-        <div class="features__icon">
-          <svg>
-            <use xlink:href="img/icons.svg#icon-trending-up"></use>
-          </svg>
+        <div class="operations__content operations__content--1">
+          <div class="operations__icon operations__icon--1">
+            <svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 576 512">
```

Comment:
Please extract the svg in a separate file and use it with the <img src=""> tag in HTML.